src/controllers/lucky.ts

```diff
@@ -0,0 +1,30 @@
+import { Request, Response } from "express";
+import * as Services from "../dal/lucky";
+import { getText } from "../helpers/arrayProcessing";
+import { getTodayAllSignRecordsFromDB } from "../dal/zodiacdeep";
+
+export async function getLuckyTextByRandomNumber(
+  req: Request,
+  res: Response
+): Promise<void> {
+  {
+    const luckyText = req?.query?.lucky;
+    let totalResult = "";
+    if (typeof luckyText !== "undefined" && luckyText === "lucky") {
```

Comment:
I'd say remove the query. By naming I'm left with the impression that this method will return only lucky results, but it is returning the standard records from the database if there is no query.

I'd say remove the query and just keep only the logic on row 15.

Reply:
done